### DISCUSSION TITLE
Were?

### DISCUSSION THREAD

**Discussion by Coolmathgamefactory** [niklas2902/py4godot#69] [Q&A]
Were do I paste these?:
$ git submodule init
$ git submodule update

**Comment by niklas2902** [comment]
Hey,
you can just execute it in the shell or git bash, depending on you system.
It should look something like this in the end:
![grafik](https://github.com/user-attachments/assets/5caea3cf-52d0-4a96-8c63-a77209afaaab)

If you have further questions, feel free to ask
